includes/editor.php

```diff
@@ -524,6 +524,8 @@ public function enqueue_scripts() {
 			'docs_elementor_site' => 'https://go.elementor.com/docs/',
 			'help_the_content_url' => 'https://go.elementor.com/the-content-missing/',
 			'help_preview_error_url' => 'https://go.elementor.com/preview-not-loaded/',
+			'help_preview_http_error_url' => 'https://go.elementor.com/preview-not-loaded/#permissions',
```

Comment:
Move all preview related config - to `config.preview`

Reply:
done